src/transfer.ts

```diff
@@ -74,3 +74,53 @@ export async function ensureFunded(funding: EnsureFundedParams, algod: Algodv2,
 
   return undefined
 }
+
+interface TransferAssetParams extends SendTransactionParams {
+  senderAccount: SendTransactionFrom
+  receiverAccount: Account
+  assetId: number
+  amount: number | bigint
+  transactionParams?: SuggestedParams
+  clawbackFrom?: SendTransactionFrom
```

Comment:
This should be `SendTransactionFrom | string` since we just need the address so you should be able to pass in a string address without needing the underlying account.